deploy/olm-catalog/service-binding-operator/service-binding-operator.package.yaml

```diff
@@ -1,5 +1,7 @@
 channels:
 - currentCSV: service-binding-operator.vBUNDLE_VERSION
+  name: alpha-0.2
+- currentCSV: service-binding-operator.v0.1.1-364
```

Comment:
I think you need to revert this changes.

Reply:
Let me know if these changes are fine. ce8ced142aed6e54659609cf8f25031ef4a4c575